### PR TITLE
lint rule for warning when e2e test does not contain axeCheck

### PR DESCRIPTION
**Pull request by Clark-W-Griswold**
## Description
When an e2e test does not contain at least 1 instance of axeCheck, we will throw a warning to alert the user to the helper method. Run rule in circle ci.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -179,6 +179,14 @@
         "no-unused-expressions": 0,
         "react/no-find-dom-node": 0
       }
+    },
+    {
+      "files": [
+        "**/*.e2e.spec.js"
```
Comment (jhonnyoddball) [inline .eslintrc:185]: Do we need "**/*.e2e.spec.jsx"?

File: script/eslint-plugin-va/rules/axe-e2e-tests.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      }
+    };
+
+    return {
+      MemberExpression: node => {
+        if (node.property.name === 'axeCheck') {
+          axeCheckCount++;
+        }
+      },
+      'Program:exit': checkAxeCountForReport,
```
Comment (jhonnyoddball) [inline script/eslint-plugin-va/rules/axe-e2e-tests.js:32]: Where is the message being displayed? at the bottom? can you attach a screenshot of the results/tests?

**Review by 1Copenut — APPROVED** [review]
LGTM. I reviewed the CI and saw the axe check in action correctly. We got a slot on the engineering demo to show our rule to a wide audience next week.

File: .eslintrc (hunk 1 above)
Comment (Clark-W-Griswold) [reply .eslintrc:185]: We only want to run it for the e2e tests currently. Open to doing this another way if there is a better way to do it. :)

File: script/eslint-plugin-va/rules/axe-e2e-tests.js (hunk 2 above)
Comment (Clark-W-Griswold) [reply script/eslint-plugin-va/rules/axe-e2e-tests.js:32]: image

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**